Doc/library/compileall.rst

```diff
@@ -193,6 +198,9 @@ Public functions
    the ``-s``, ``-p`` and ``-e`` options described above.
    They may be specified as ``str``, ``bytes`` or :py:class:`os.PathLike`.
 
+   If *hardlink_dupes* is ``True``, hardlinks are used to prevent duplicates
```

Comment:
"is true" is more general, for code using "if flag" rather than "if flag is True:":

```suggestion
   If *hardlink_dupes* is true, hardlinks are used to prevent duplicates
```

Same remark for compile_file documentation.